RecoHGCal/TICL/plugins/HGCDoublet.cc

```diff
@@ -26,6 +26,9 @@ bool HGCDoublet::checkCompatibilityAndTag(std::vector<HGCDoublet> &allDoublets,
       yi[j] = otherDoublet.innerY();
       zi[j] = otherDoublet.innerZ();
       seedi[j] = otherDoublet.seedIndex();
+      if (debug) {
```

Comment:
I wonder why there should be both the ```if (debug)```, and the ```LogDebug```: since the ```LogDebug``` has to be enabled anyhow, I woud remove the further request based on the ```debug``` flag.
(Not specific for this PR, since it is the same in the whole file...)

Reply:
The reason is very easy. `LogDebug` configuration is a pain with the `MessageLogger`.
Debugging is so much easier with a simple:

```c++
#define LogDebug(X) std::cout << X
```

at the beginning of the file and a boolean set at the configuration level.
Unless there are strong concern about that, I'd kindly ask to leave it as is.

Reply:
If you redefine ```LogDebug``` at the beginning of the file, I don't see the reason for also having a bool  in addition to steer it. On the other hand, I think this is a (less than) second order issue, fine with me to leave it as such.